### PR TITLE
Download file

### PR DESCRIPTION
**Pull request by peeyush14goyal**
Added a Download button for downloading the files. Also Centralized the **loading icon**
image

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -55,6 +55,10 @@
       crossorigin="use-credentials"
     />
     <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <!-- <meta
+      http-equiv="Content-Security-Policy"
+      content="frame-src https://care-patient-staging.s3.amazonaws.com/"
+    /> -->
```
Comment (bodhish) [inline src/index.html]: @peeyush14goyal can you remove this

File: src/Components/Patient/FileUpload.tsx [hunk 2]
```diff
@@ -36,6 +36,7 @@ import { VoiceRecorder } from "../../Utils/VoiceRecorder";
 import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";
 import Modal from "@material-ui/core/Modal";
 import { Close } from "@material-ui/icons";
+import { BrowserClient } from "@sentry/browser";
```
Comment (bodhish) [inline src/Components/Patient/FileUpload.tsx]: Unused Import
```suggestion

```

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Patient/FileUpload.tsx (hunk 2 above)
Comment (peeyush14goyal) [reply src/Components/Patient/FileUpload.tsx]: ok. Changes done @bodhish

File: src/index.html (hunk 1 above)
Comment (peeyush14goyal) [reply src/index.html]: Done @bodhish

**Review by bodhish — APPROVED** [review]
LGTM

**Pull request merged**